Kitodo/src/main/java/org/kitodo/production/thread/ImportEadProcessesThread.java

```diff
@@ -0,0 +1,348 @@
+                        } else {
+                            if (inParentProcessElement || inProcessElement) {
+                                String content = event.toString();
+                                if (StringUtils.isNotBlank(content)) {
+                                    stringBuilder.append(removeDefaultNamespaceUri(content));
+                                }
+                            }
+                        }
+                        break;
+                    case XMLEvent.END_ELEMENT:
+                        EndElement endElement = event.asEndElement();
+                        String endElementName = endElement.getName().getLocalPart();
+                        if (inProcessElement && StringConstants.C_TAG_NAME.equals(endElementName)) {
+                            int progress = (count * 100) / numberOfElements;
+                            setProgress(progress);
+                            inProcessElement = false;
+                            String content = event.toString();
+                            stringBuilder.append(removeDefaultNamespaceUri(content));
+                            newProcessIds.add(parseXmlStringToProcessedTempProcess(stringBuilder.toString()).getProcess().getId());
+                            stringBuilder = new StringBuilder();
+                        } else {
+                            if (inParentProcessElement || inProcessElement) {
+                                String content = event.toString();
+                                if (StringUtils.isNotBlank(content)) {
+                                    stringBuilder.append(removeDefaultNamespaceUri(content));
+                                }
+                            }
+                        }
+                        break;
+                    default:
+                        if (inParentProcessElement || inProcessElement) {
+                            String content = event.toString();
+                            if (StringUtils.isNotBlank(content)) {
+                                stringBuilder.append(removeDefaultNamespaceUri(content));
+                            }
+                        }
+                        break;
+                }
+            }
+
+        } catch (XMLStreamException | IOException | ParserConfigurationException | SAXException
+                 | UnsupportedFormatException | XPathExpressionException | ProcessGenerationException
+                 | URISyntaxException | InvalidMetadataValueException | TransformerException
+                 | NoSuchMetadataFieldException | DataException | CommandException e) {
+            logger.error(e.getMessage(), e);
+            cleanUpProcesses(newProcessIds, newParentId);
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void cleanUpProcesses(List<Integer> processIds, int newParentId) {
+        logger.error("Deleting processes created until this point to resolve erroneous intermediate state.");
```

Comment:
Is `error` log level correct? Should not log level `info` or lower be used here? It did not look like that an error occur and get logged.